test/functional/p2p_ibd.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test inv messages and fee filters during and after IBD."""
+
+from decimal import Decimal
+
+from test_framework.messages import COIN
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+MAX_FEE_FILTER = Decimal(str(9170997 / COIN))
```

Comment:
Seems a bit odd to use floating point, but then put the result in a decimal. Why not `Decimal(9170997)/COIN`?